src/App.js

```diff
@@ -1,26 +1,68 @@
-import './App.css';
-import Counter from './Components/Counter/Counter';
-import SearchForm from './Components/SearchForm/SearchForm';
+import React, {useState} from "react";
+import './App.scss';
 import GenreSelector from './Components/GenreSelector/GenreSelector';
-import { GenreListArray } from './data';
+import SortControl from './Components/SortControl/SortControl';
+import ColorPalette from './Components/ColorPalette/ColorPalette';
+import MoviesFound from './Components/MoviesFound/MoviesFound';
+import MovieTile from './Components/MovieTile/MovieTile';
+import Logo from './Components/Logo/Logo';
+import Header from './Components/Header/Header';
+import MovieDetails from "./Components/MovieDetails/MovieDetails";
+import {GenreListArray, SortControlArray, movieTilesArray} from './data';
+
+export function SwitchComponents({ active, children }) {
```

Comment:
You can keep it here, but usually in React is good practice to keep only one component per file